app/controllers/admin/financial_reports_controller.rb

```diff
@@ -1,6 +1,9 @@
 class Admin::FinancialReportsController < Admin::BaseController
   before_action :load_organisation
   before_action :load_financial_report, only: %i[edit update destroy]
+  layout :get_layout
+
+  def index; end
```

Comment:
I think we're missing the bootstrap implementation here. So this needs to be 

```
def index
  render_design_system(:index, :legacy_index)
end

```

And we need to retain the boostrap implementation in the `legact_index` view. Otherwise the user will always see the design system page.